roofit/roofitcore/src/Initialisation.cxx

```diff
@@ -62,25 +69,31 @@ void loadComputeLibrary() {
   }
 
 #if __GNUC__ > 5 || defined(__clang__)
-  if (__builtin_cpu_supports("avx512f")) {
-    libSuffix = "_AVX512f";
+  //skylake-avx512 support
+  if (__builtin_cpu_supports("avx512cd") && __builtin_cpu_supports("avx512vl") && __builtin_cpu_supports("avx512bw") && __builtin_cpu_supports("avx512dq"))  {
+    libSuffix = "_AVX512";
   }
 #endif
 
-#endif
 
   for (auto&& libName : libNames) {
     libName += libSuffix;
     const auto returnValue = gSystem->Load(libName.c_str());
 
     if (returnValue == -1 || returnValue == -2) {
-      throw std::runtime_error("RooFit was unable to load its computation library " + libName);
+      std::cout << "RooFit was unable to load its computation library " + libName << std::endl;
```

Comment:
How's that not an error? I.e. `std::cerr`? And here, too - please make sure this shouldn't use some logging framework. O, and please consider adding *where* this occurs, so people can pinpoint this to us, e.g. `"Error in RooFit::loadComputeLibrary(): ..."` (or whatever the fully qualified name of that function is)

Reply:
I will not try to load the library if suffix is empty